packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -29,42 +31,20 @@ interface ModalState {
   modalContainer: HTMLDivElement | null;
 }
 
-export interface ModalActionsProps {
-  withBorder?: boolean;
-}
-
-export interface ModalHeaderProps {
-  withBorder?: boolean;
+interface Action extends Omit<ButtonProps, 'children'> {
+  text?: string;
```

Comment:
Should this be required?

Reply:
I was thinking we could have only icon buttons

Reply:
Good point, can't think of it being a good use case but let's leave it for now. Good catch.